test/box/gh-6203-fiber-join-with-timeout.result

```diff
@@ -0,0 +1,55 @@
+env = require('test_run')
```

Comment:
The test should be in app, because it has nothing to do with box.

Reply:
ok, moved it to app